umh-core/Makefile

```
@@ -340,22 +338,10 @@ nilaway:
 		-exclude-errors-in-files="integration/" \
 		./...
 
-betteralign:
-	betteralign -test=false ./... || (echo "Consider running 'make betteralign-fix' to fix the issues" && exit 1)
-betteralign-fix:
-	betteralign -test=false -apply ./...
+# Meta target for vet, nilaway and lint
+vet-nilaway-lint: vet nilaway lint
```

checkmake (0.3.2)

[warning] Target "vet-nilaway-lint" should be declared PHONY.

(phonydeclared)